bin/test-workflow/stop

```diff
@@ -3,15 +3,22 @@ set -uo pipefail
 
 source utils.sh
 
+uninstall_helm_charts="
+helm uninstall cluster-prep-$UNIQUE_TEST_ID -n $CONJUR_NAMESPACE_NAME &&
```

Comment:
This script has a whole bunch of env variables containing strings that aren't encapsulated in quotes.
Would you mind adding the quotes in while we're making changes here?